src/models.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID as sa_uuid
+from sqlalchemy.orm import relationship
+
+from src.service import Base
+
+
+class Projects(Base):
+    __tablename__ = "projects"
+
+    id = sa.Column(sa_uuid(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    name = sa.Column(sa.String, nullable=False)
+    description = sa.Column(sa.String, nullable=False)
```

Comment:
again, in pydantic schema you allow it to be nullable, here it is not. You will have conflicts. Please choose one approach